clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -53,6 +53,7 @@
 #include <iostream>
 #include <string>
 #include <sys/syscall.h>
+#include <sys/types.h>
```

Comment:
So this test must already assume it's only built and run for Linux: would be good to add `#ifndef LINUX #error Only Linux is supported #endif` to make that clear to both human readers of the code and future porters to non-Linux via clear error messages.